hphp/zend/zend-strtod.cpp

```diff
@@ -1191,7 +1191,7 @@ static CONST double tinytens[] = { 1e-16, 1e-32 };
 #endif
 
 
-static int quorem(Bigint *b, Bigint *S)
```

Comment:
FWIW this is 'quotient remainder', although it confused me too :)

Reply:
Doh. I kinda wondered if it might be something like that. But the total lack of a comment before the function left me puzzled (c.f. `Generator::getMarkedCollectable`). I figured a reviewer would set me straight if I was wrong.

Renaming the function to `quoRem` or something would help readers.